components/apps.js

```diff
@@ -0,0 +1,32 @@
+class App {
+  handleGetGradesError(error) {
+    console.error("error");
+  }
+
+  handleGetGradesSuccess(grades) {
+    this.gradeTable.updateGrades(grades);
+  }
+
+  constructor (gradeTable) {
```

Comment:
Usually the constuctor should be the first method in a class. You dont need to change it now, but before the next pr you should change it